### ISSUE TITLE
Beyond the Grave quest

### ISSUE THREAD

**Issue by InsertnamespaceHere** [Wynncraft/Issues#7460]
Hello I seem to be stuck in the beyond the grave quest, in the part when you solve the "TOMBSTONE" Puzzle and the dialog appears and the glass shatters happens, there is only a glass block and fire in the ground but the window isn't broken, after a while the fire will disappear and everything but I can never exit the window

**Comment by thecoolkogut4** [comment]
what happens if you try switching servers? are you go8ng to the right window?

**Comment by InsertnamespaceHere** [comment]
The Same thing, I've tried switching servers 3 times, chaging class and looking through tutorials and its the smae spot i am at

**Comment by microninja123** [comment]
the same thing is happening to me, only one glass breaks on the window.
![2017-04-30_22 10 51](https://cloud.githubusercontent.com/assets/28247528/25572664/31878b80-2df2-11e7-951d-3c5d3bb22e30.png)

**Comment by Dalarn** [comment]
Yes this happened to me yesterday after I typed in END into the puzzle. Now the window won't break no matter what I try. I even went in a group effort to finish the quest. We completed the puzzle and the "window shattered message" appeared as always but only my teammates could go through the window. I need mod/admin assistance please. I would like to finish this quest. :)

My ign: DalarnRagei

**Comment by Dalarn** [comment]
Ok ill just wait for the quest to be fixed/ or reset.
